pages/Documents/MessagingChannels/AppleBusinessChatSDK/installation.md

```diff
@@ -4,8 +4,8 @@ redirect_from:
 Keywords:
 sitesection: Documents
 categoryname: "Messaging Channels"
-documentname: Apple Business Chat SDK
-permalink: apple-business-chat-sdk-installation.html
+documentname: Apple Messages for Business SDK
+permalink: apple-messages-for-business-sdk-installation.html
```

Comment:
Did you test locally? IIRC the `documentname` needs updating in the documentsupdated.yaml (where the diff doesn’t show a change like this).

Also, please add a redirect to make sure the old path redirects here—this can be done through `redirect_from` ([example](https://github.com/LivePersonInc/developer-center/blob/master/pages/Documents/APIGuidelines/getting-started-with-authentication.md)).

Reply:
reverted -- will address page names and redirects separately